flow/flow.go

```diff
@@ -876,6 +876,74 @@ func (f *Flow) newTransportLayer(packet *Packet, opts Opts) error {
 	return nil
 }
 
+func convertToFlowDNSResourceRecord(rr *layers.DNSResourceRecord) fl.DNSResourceRecord {
+	var SOA *fl.DNSSOA
+	var SRV *fl.DNSSRV
+	var MX *fl.DNSMX
+	var TXTs []string
+	var TXT []byte
+	var NS, CNAME, PTR, IP string
+	switch rr.Type {
+	case layers.DNSTypeA:
+		IP = rr.IP.String()
```

Comment:
```
case layers.DNSTypeA, layers.DNSTypeAAAA:
return  fl.DNSResourceRecord{IP: rr.IP.String()}
```
and so on for the rest of the types instead of returning a big "almost" empty struct at the and of the function

Reply:
done